venues/thecvf.com/ECCV/2020/Conference/python/post-custom-load-edges.py

```diff
@@ -1,130 +1,129 @@
 import openreview
-import csv
+import argparse
 from tqdm import tqdm
 
-client = openreview.Client(baseurl = 'https://openreview.net')
+if __name__ == '__main__':
+    ## Argument handling
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--baseurl', help="base url")
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+    args = parser.parse_args()
+
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+    reviewer_group_name = 'Reviewers'
```

Comment:
I'm not sure why this variable exists

Reply:
It was added initially when I was unsure if we would need to post custom edges for ACs as well. But now that we know about this I can change this.

Reply:
Fixed.